harness/determined/keras/_tf_keras_context.py

```diff
@@ -35,6 +35,33 @@ def __init__(self, env: det.EnvContext, hvd_config: horovod.HorovodContext):
         self.compile_args = None  # type: Optional[inspect.BoundArguments]
         self.train_config = None  # type: Optional[TFKerasTrainConfig]
 
+        # The following attributes may be configured via _configure_fit().  Defaults match the
+        # normal keras.fit() defaults.
+        self._fit_verbose = True
+        self._fit_shuffle = True
+        self._fit_workers = 1
+        self._fit_use_multiprocessing = False
+        self._fit_max_queue_size = 10
+
+    def _configure_fit(
```

Comment:
question: is this just for native ?

Reply:
no, this is the public api improvement I mentioned in ml-ag